x-pack/legacy/plugins/siem/public/components/formatted_date/index.test.tsx

```diff
@@ -6,174 +6,166 @@
-        const wrapper = mount(
-          <FormattedDate fieldName="@timestamp" value={'2019-05-28T22:04:49.957Z'} />
-        );
-        expect(wrapper.text()).toEqual('May 28, 2019 @ 22:04:49.957');
-      });
-
-      test('it renders against an empty string as an empty string placeholder', () => {
-        mockUseUiSetting$.mockImplementation(getMockKibanaUiSetting(mockFrameworks.default_UTC));
-        const wrapper = mount(
-          <TestProviders>
-            <FormattedDate fieldName="@timestamp" value={''} />
-          </TestProviders>
-        );
-        expect(wrapper.text()).toEqual('(Empty String)');
-      });
-
-      test('it renders against an null as a EMPTY_VALUE', () => {
-        mockUseUiSetting$.mockImplementation(getMockKibanaUiSetting(mockFrameworks.default_UTC));
-        const wrapper = mount(
-          <TestProviders>
-            <FormattedDate fieldName="@timestamp" value={null} />
-          </TestProviders>
-        );
-        expect(wrapper.text()).toEqual(getEmptyValue());
-      });
-
-      test('it renders against an undefined as a EMPTY_VALUE', () => {
-        mockUseUiSetting$.mockImplementation(getMockKibanaUiSetting(mockFrameworks.default_UTC));
-        const wrapper = mount(
-          <TestProviders>
-            <FormattedDate fieldName="@timestamp" value={undefined} />
-          </TestProviders>
-        );
-        expect(wrapper.text()).toEqual(getEmptyValue());
-      });
-
-      test('it renders against an invalid date time as just the string its self', () => {
-        mockUseUiSetting$.mockImplementation(getMockKibanaUiSetting(mockFrameworks.default_UTC));
-        const wrapper = mount(
-          <TestProviders>
-            <FormattedDate fieldName="@timestamp" value={'Rebecca Evan Braden'} />
-          </TestProviders>
-        );
-        expect(wrapper.text()).toEqual('Rebecca Evan Braden');
-      });
+    test('it renders against a numeric epoch', () => {
+      const wrapper = mount(<FormattedDate fieldName="@timestamp" value={1559079339000} />);
+      expect(wrapper.text()).toEqual('May 28, 2019 @ 21:35:39.000');
+    });
+
+    test('it renders against a string epoch', () => {
+      const wrapper = mount(<FormattedDate fieldName="@timestamp" value={'1559079339000'} />);
+      expect(wrapper.text()).toEqual('May 28, 2019 @ 21:35:39.000');
+    });
+
+    test('it renders against a ISO string', () => {
+      const wrapper = mount(
+        <FormattedDate fieldName="@timestamp" value={'2019-05-28T22:04:49.957Z'} />
+      );
+      expect(wrapper.text()).toEqual('May 28, 2019 @ 22:04:49.957');
+    });
+
+    test('it renders against an empty string as an empty string placeholder', () => {
+      const wrapper = mount(
+        <TestProviders>
+          <FormattedDate fieldName="@timestamp" value={''} />
+        </TestProviders>
+      );
+
+      expect(wrapper.text()).toEqual('(Empty String)');
```

Comment:
Optional:
There's a placeholder constant for this already in this file of `getEmptyString()`:

```ts
expect(wrapper.text()).toEqual(getEmptyString()));
```

I would use it here to make the test more resilient against constant name changes such as we want a different name than `(Empty String)` in the future.

Reply:
fixed!